src/vaults/apys/implementations/convex.utils.js

```diff
@@ -2,6 +2,7 @@ const BigNumber = require('bignumber.js')
 const { token, crvYPool, pool } = require('../../../lib/web3/contracts')
 const { getTokenPriceByAddress } = require('../../../prices/coingecko.js')
 const { web3 } = require('../../../lib/web3')
+const addresses = require('../../../../data/mainnet/addresses.json')
```

Comment:
Use `../../../lib/data/addresses.json` here and the necessary addresses to that file.

Reply:
done